packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx

```diff
@@ -160,7 +160,7 @@ export const RecordInlineCellContainer = ({
               content={label}
               clickable
               noArrow
-              place="left"
+              place="bottom"
```

Comment:
This change seems unrelated to the PR
@Bonapara what do you think should the tool tip be below or on left of the hoverable label?

Reply:
No strong opinion, but maybe below is more responsive?

Reply:
Actually for left the tool tip was overlaying sidebar's text